saleor/static/dashboard-next/siteSettings/components/SiteSettingsPage/SiteSettingsPage.tsx

```diff
@@ -12,13 +12,25 @@ import i18n from "../../../i18n";
 import { maybe } from "../../../misc";
 import { AuthorizationKeyType } from "../../../types/globalTypes";
 import { SiteSettings_shop } from "../../types/SiteSettings";
+import SiteSettingsAddress from "../SiteSettingsAddress/SiteSettingsAddress";
 import SiteSettingsDetails from "../SiteSettingsDetails/SiteSettingsDetails";
 import SiteSettingsKeys from "../SiteSettingsKeys/SiteSettingsKeys";
 
 export interface SiteSettingsPageFormData {
+  city: string;
```

Comment:
It would be cleaner to extract address related fields to `SiteSettingsPageAddressFormData` and then use `extends` in `SiteSettingsPageFormData`